doc/introduction/measurements.rst

```diff
@@ -77,7 +77,7 @@ by a ``Hadamard`` and ``CNOT`` gate.
     import pennylane as qml
     from pennylane import numpy as np
 
-    dev = qml.device("default.qubit", wires=2)
+    dev = qml.device("default.qubit", wires=2, shots=1)
```

Comment:
```suggestion
    dev = qml.device("default.qubit", wires=2, shots=1000)
```

Below it mentions "1000 samples per observable" and a shape of `(2,1000)`.

Reply:
Thanks :)